### ISSUE TITLE
slim examples don't provide coders with self-adaption batch_size, and now i must set batch_size value in advance

### ISSUE THREAD

**Issue by ronyuzhang** [tensorflow/models#2947]

**Comment by ronyuzhang** [comment]
python train_image_classifier_faces.py \
  --train_dir=${TRAIN_DIR} \
  --dataset_name=faces \
  --dataset_split_name=train \
  --dataset_dir=${DATASET_DIR} \
  --model_name=inception_resnet_v1 \
  --checkpoint_path=${CKPT_PATH} \
  --checkpoint_exclude_scopes=InceptionResnetV1/Logits,InceptionResnetV1/Bottleneck,Logits \
  --trainable_scopes=InceptionResnetV1/Logits,InceptionResnetV1/Bottleneck,Logits \
  --max_number_of_steps=200000 \
 #  --batch_size=32 \
  --learning_rate=0.01 \
  --learning_rate_decay_type=fixed \
  --save_interval_secs=1000 \
  --save_summaries_secs=60 \
  --log_every_n_steps=100 \
  --optimizer=rmsprop \
  --weight_decay=0.00004 \
  --center_loss_factor=1e-2 \
  --center_loss_alfa=0.9

**Comment by ronyuzhang** [comment]
@bmabey @alextp @moonboots @ewilderj @jtoy

**Comment by reedwm** [comment]
I apologize but I am having a hard time understanding what the problem is, where the problem is, and what version it affects. Please resubmit and pay attention to the issue template (https://github.com/tensorflow/tensorflow/issues/new) . Please provide all the information it asks. Thank you.

**Comment by ronyuzhang** [comment]
@reedwm  sorry I don't explain away。 solution as follows：
 I set batch size = 32 on training step. and I have to set batch size =32, or I'll make a mistake on evaluation step.

**Comment by reedwm** [comment]
@ronyuzhang, can you fill out the issue template that appears when you click [New issue](https://github.com/tensorflow/models/issues/new)? Thank you.

**Comment by wt-huang** [comment]
Closing this issue due to inactivity, feel free to reopen if this is still an issue.
